book/course/spack/modules.md

```diff
@@ -0,0 +1,71 @@
+# Adjusting how we name modules
+
+In the last section we built a version of pigz, and loaded it:
+
+```bash
+$ module add pigz-2.7-gcc-8.5.0-qkx6pxv
+$ pigz --version
+pigz 2.7
+```
+
+Now, it's very definitive what you're loading (pigz version 2.7 built with gcc
+8.5.0 with a hash to pin it down further), but it's a bit wordy.  This format
```

Comment:
Are we assuming the audience understand what a `hash` is here? Do we need to know there's a hash or could we just say it's got a big long unwieldy module name?

Not sure of the right answer just trying to balance details v. cognitive load for the audience

Reply:
You sort of touch on this later on, so inclined to leave it as is

Reply:
I think it's still worth giving a basic explanation of what a hash is (a unique identifier for this particular version of the software).